content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -718,3 +718,60 @@ Clean up the Pod when you're finished with it:
 ```shell
 kubectl delete pod myapp
 ```
+
+### Custom Profile {#custom-profile}
```

Comment:
```suggestion
## Custom profiles for ephemeral containers {#custom-profile}
```

Reply:
Same as Debugging Profiles above, custom profile can be applied not only to an ephemeral containers, but also to a copied Pods or a debugging pods to debug nodes.
Although the following case uses an ephemeral container as an example, I think the title should remain more general, as it is now.
(While other examples can be provided, it would be redundant)

Furthermore, I think custom profile is an extension of debugging profile.
So it would be better for the Custom Profile section under Debugging Profiles to be at level `###` instead of `##`, or to add a new comprehensive section that consolidates the Debug Profile and Custom Profile.